docs/site/sidebars/lb4_sidebar.yml

```
@@ -664,6 +664,10 @@ children:
         url: Cloudant-connector.html
         output: 'web, pdf'
 
+      - title: 'CouchDB 2 connector'
```

I think it's clear to just have `CouchDB connector`
```suggestion
      - title: 'CouchDB connector'
```